tensorflow/core/BUILD

```diff
@@ -156,6 +156,8 @@ load(
 )
 load("@local_config_cuda//cuda:build_defs.bzl", "if_cuda")
 load("@local_config_cuda//cuda:build_defs.bzl", "if_cuda_is_configured")
+load("@local_config_rocm//rocm:build_defs.bzl", "if_rocm")
```

Comment:
These macros are not used, please remove.

Reply:
done.

also rebased the commits to squash all the code review related update commits into one